index.html

```diff
@@ -69,7 +76,7 @@
 or benefit is commonplace, when the technology is available as part of the web
 platform.
 
-These questions are an accompaniment to the [[EWP|Ethical Web Principles]],
+These questions are an accompaniment to the [[ethical-web-principles|Ethical Web Principles]],
```

Comment:
You should be able to get this with

```suggestion
These questions are an accompaniment to the [[[ethical-web-principles]]],
```